pkg/splunk/enterprise/licensemaster_test.go

```diff
@@ -38,6 +38,7 @@ func TestApplyLicenseMaster(t *testing.T) {
 		{MetaName: "*v1.Secret-test-splunk-stack1-license-master-secret-v1"},
 		{MetaName: "*v1.ConfigMap-test-splunk-stack1-licensemaster-app-list"},
 		{MetaName: "*v1.StatefulSet-test-splunk-stack1-license-master"},
+		{MetaName: "*v1.StatefulSet-test-splunk-stack1-license-master"},
```

Comment:
Why is this func call added again? Same question for `pkg/splunk/enterprise/searchheadcluster_test.go` and `pkg/splunk/enterprise/standalone_test.go`

Reply:
validateMonitoringConsoleRef function does a GET call to pull the statefulset like below, so one more call added
```
var current appsv1.StatefulSet
err = c.Get(context.TODO(), namespacedName, &current)
```